savant/deepstream/buffer_processor.py

```diff
@@ -694,6 +695,19 @@ def prepare_element_output(self, element: PipelineElement, buffer: Gst.Buffer):
                                 obj_label = build_model_object_key(
                                     element.name, obj.label
                                 )
+                                obj_cls_id = MERGED_CLASSES[element.name].get(
```

Comment:
Are you sure there is no option for element.name not to be represented in MERGED_CLASSES?

Reply:
Should be ok, as the `MERGED_CLASSES` is a defaultdict